nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-services/src/main/java/org/apache/nifi/accumulo/controllerservices/AccumuloService.java

```diff
@@ -50,52 +61,95 @@
  */
 @RequiresInstanceClassLoading
 @Tags({"accumulo", "client", "service"})
-@CapabilityDescription("A controller service for accessing an HBase client.")
+@CapabilityDescription("A controller service for accessing an Accumulo Client.")
 public class AccumuloService extends AbstractControllerService implements BaseAccumuloService {
 
-    private enum AuthenticationType{
+    private enum AuthenticationType {
         PASSWORD,
+        KERBEROS,
         NONE
     }
 
     protected static final PropertyDescriptor ZOOKEEPER_QUORUM = new PropertyDescriptor.Builder()
-            .name("ZooKeeper Quorum")
+            .name("accumulo-zooKeeper-quorum")
```

Comment:
We can't change the name of existing properties, that is the reason for displayName which can be used to change what the user sees in the UI. The regular name is a key that is used in the flow configuration, so if we change it then anyone using this component will have to reconfigure it when they upgrade to the release with this change because their will flow will reference a now unknown property name "ZooKeeper Quorum".

Reply:
Thanks @bbende restored the name to original